io_scene_niftools/modules/nif_import/collision/havok.py

```diff
@@ -62,7 +62,7 @@ def __init__(self):
         collision.DICT_HAVOK_OBJECTS = {}
 
         # TODO [collision][havok][property] Need better way to set this, maybe user property
-        if bpy.context.scene.niftools_scene.user_version == 12 and bpy.context.scene.niftools_scene.user_version_2 == 83:
+        if bpy.context.scene.niftools_scene.user_version == 12 and bpy.context.scene.niftools_scene.user_version_2 >= 83:
```

Comment:
another candidate for helper function like use_special_havok_scale()

Reply:
I should probably just use the havok scale on the nif file object, since that gets calculated there anyways and get rid of the two different places where the havok scale is stored in the addon.

Reply:
Sounds good too!